src/Orleans.Core.Abstractions/Orleans.Core.Abstractions.csproj

```diff
@@ -0,0 +1,15 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <PackageId>Orleans.Core.Abstractions</PackageId>
+    <Title>Microsoft Orleans Core Abstractions</Title>
+    <Description>Core abstractions library of Microsoft Orleans</Description>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <AssemblyName>Orleans.Core.Abstractions</AssemblyName>
+    <RootNamespace>Orleans</RootNamespace>
+    <TargetFramework>netstandard1.5</TargetFramework>
```

Comment:
Doesn't it work if it's netstandard1.3? This is so that end users don't require new tooling to reference it, and since there will be no logic, it probably doesn't require anything from either 1.5 or 1.6, which are the biggest offenders in the new .NET Standard world.